docs/documentation/test/eksempler/melding/eksempelMedAlleTilfeller.xml

```diff
@@ -17,12 +17,12 @@
 		<fastsattMerverdiavgift>77511</fastsattMerverdiavgift>
 		<mvaSpesifikasjonslinje>
 			<mvaKode>1</mvaKode>
-			<mvaKodeRegnskapsystem>25% inng mva</mvaKodeRegnskapsystem>
```

Comment:
Vi skulle ikke escapet i stedet for å fjerne?

Reply:
Hva mener du med "escapet"? Fordi, dessverre feilen kommer med f.eks. 25\% eller alle mulig encoding formater

Reply:
Men i prinsipp, jeg er i enig at kanskje eksempel filer skulle vise en "riktig" eksempel istedenfor en som er endret så at det funker med vårt nåværende teknisk gjeld